### ISSUE TITLE
What is OPMode and can be implemented to the game for solving extrapolation issues?

### ISSUE THREAD

**Issue by Mirac35** [haxball/haxball-issues#1758]
Title. There are some flickering players going on lately and they're talking about OPMode hack or something. Modifications seem clientside but affecting everyone in the room but the user/hacker. Maybe updating game with OPMode and forced use of it/applying to global could solve some extrapolation related problems. IDK.

**Comment by furkanardagul** [comment]
You can use kick rate, which decodes the macro, but it won't do much. Since these cheats are user-sided, it does not seem possible to prevent them. If he has a clear idea, I will be glad if he informs me.

**Comment by Mirac35** [comment]
> You can use kick rate, which decodes the macro, but it won't do much. Since these cheats are user-sided, it does not seem possible to prevent them. If he has a clear idea, I will be glad if he informs me.

Yup, i got the chance using opmode 'cheat' and theres no negative downsides. had to reduce my extrapolation from 135 to 80 to play the game without any client-side flickering visual issues. i hope this codes will be added to the core and enhance extrapolation feature itself.

**Comment by wxyz-abcd** [comment]
I'm sometimes working on a solution to that cheat, but it's still not ready yet.

**Comment by Mirac35** [comment]
@wxyz-abcd chu mean by solution? neutralizing  it or removing the "cheat" tag?
@basro i think we're onto something here.
